src/Operation/CreateIndexes.php

```diff
@@ -89,6 +100,8 @@ public function execute(Server $server)
      * command.
      *
      * @param Server $server
+     *
+     * @return mixed
```

Comment:
I realized while reviewing that this function's return value is never used. I've removed it entirely in https://github.com/mongodb/mongo-php-library/pull/188.